pkg/services/apiserver/builder/helper.go

```diff
@@ -36,8 +36,13 @@ import (
 	"github.com/grafana/grafana/pkg/storage/unified/apistore"
 )
 
+type BuildHandlerChainFuncFromBuilders = func([]APIGroupBuilder) BuildHandlerChainFunc
 type BuildHandlerChainFunc = func(delegateHandler http.Handler, c *genericapiserver.Config) http.Handler
 
+func ProvideDefaultBuildHandlerChainFuncFromBuilders() BuildHandlerChainFuncFromBuilders {
```

Comment:
Providing the default on the wire as any overrides in enterprise will also use the same method to automagically transform the APIServer's build chain.